app/views/courses/show.html.erb

```diff
@@ -73,18 +73,18 @@
 <div class="course-tabs govuk-tabs">
   <h2 class="govuk-tabs__title">Contents</h2>
 
-  <ul class="govuk-tabs__list">
+  <ul class="govuk-tabs__list" role="tablist">
     <li class="govuk-tabs__list-item" role="presentation">
       <%=
         link_to "#description",
-        class: "govuk-tabs__tab govuk-tabs__tab--selected", id: "description", role: 'tab', tabindex: '-1', aria: { selected: true } do
+        class: "govuk-tabs__tab govuk-tabs__tab--selected", role: 'tab', tabindex: '-1', aria: { selected: true } do
       %>
         Description<br>
         <span class="govuk-body-s govuk-!-font-weight-regular pointer-events-none">Content, fees and eligibility</span>
       <% end %>
     </li>
 
-    <li class="govuk-tabs__list-item">
+    <li class="govuk-tabs__list-item" role="presentation">
```

Comment:
Why is this role needed?

It's not in the design system component, which will have been rigorously tested for accessibility: https://design-system.service.gov.uk/components/tabs/

Reply:
[I wasn't quite sure if it was necessary but it existed on the sibling `li`](https://github.com/DFE-Digital/manage-courses-frontend/blob/b209a35930b2ea2f499d7ec5db9c99ef70ed948d/app/views/courses/show.html.erb#L77)

Reply:
Probably best to remove them all

Reply:
Ah, that’s probably because it’s on the currently selected tab, where the link isn't particularly useful.

Reply:
So leave there and remove on the other?